test-case/check-audio-equalizer.sh

```diff
@@ -53,8 +53,8 @@ func_test_eq()
         return 1
     fi
     # test with aplay
-    dlogc "aplay -D$dev -f $fmt -c $channel -r $rate /dev/zero -d $duration"
-    aplay -D$dev -f $fmt -c $channel /dev/zero -d $duration
+    dlogc "$cmd -D$dev -f $fmt -c $channel -r $rate -d $duration $dummy_file"
```

Comment:
Please also change the upper description " test with aplay" as we supports arecord now.